docs/tutorials/01_introductory/viz_solar_system.py

```diff
@@ -183,7 +183,7 @@ def get_orbit_actor(orbit_points):
 # orbit actors into the scene. Also initialize the track variables for each
 # planet.
 
-orbit_points = np.zeros((2001, 3), dtype='f8')
+orbit_points = np.zeros((2200, 3), dtype='f8')
```

Comment:
Can you replace `f8` by `float`? Thank you.

Reply:
Done, and sorry about that.